### PR TITLE
non-free/yed: new aport

### PR DESCRIPTION
**Pull request by durrendal**
New aport for yEd

### CODE REVIEW

File: testing/yed/yed.install [hunk 1]
```diff
@@ -0,0 +1,13 @@
+post_upgrade() {
+			   post_install $1
+}
+
+post_install() {
+			   echo ">>> You have to read and accept the license agreement under"
+			   echo ">>> /usr/share/licenses/yed/license.html before using yEd."
```
[the remark this answers is not in the document]
Comment (durrendal) [reply testing/yed/yed.install]: Right you are, I wasn't thinking when I pushed this last night. I went ahead and moved it from testing to non-free and corrected the new-line issues as well.

File: non-free/yed/yed.install [hunk 2]
```diff
@@ -0,0 +1,12 @@
+post_upgrade() {
+			   post_install $1
+}
+
+post_install() {
+			   echo ">>> You have to read and accept the license agreement under"
+			   echo ">>> /usr/share/licenses/yed/license.html before using yEd."
+}
```
Comment (Cogitri) [inline non-free/yed/yed.install]: Why indent with so many tabs? :o

Comment (durrendal) [reply non-free/yed/yed.install]: I think Emacs just hates me. Maxice8 has gotten onto me about this before. Give me just a second and I can fix that.

Comment (durrendal) [reply non-free/yed/yed.install]: And it should be fixed now!

Comment (Cogitri) [reply non-free/yed/yed.install]: Seems like it's still like previously in post_upgrade :P

**Comment by Cogitri** [comment]
Thank you :)

**Pull request closed without merging**